packages/react-examples/.storybook/preview.js

```diff
@@ -28,7 +28,15 @@ configure(loadStories, module);
 export const parameters = {
   options: {
     storySort: {
-      order: ['Concepts/Introduction', 'Concepts/Developer', 'Concepts', 'Theme', 'Components'],
+      order: [
+        'Concepts/Introduction',
+        'Concepts/Developer',
+        'Concepts',
+        'Migrations/Flex/Overview',
```

Comment:
q: don't we wanna have migrations at the bottom ?

Reply:
Given the importance of the migrations, I felt like it was kind of lost and would lose more and more visibility the more components we added.
Should I leave it in the bottom?

Reply:
lets merge as is and we will verify during sync. thx